components/ExperiencesList/View.js

```diff
@@ -72,15 +72,15 @@ const ExperiencesListComponent = ({ clearIdentity }) => {
               <div className="text-15px text-brand-blue">Take your ID with you </div>
               <div></div>
             </div>
-            <a href="https://www.auti.sm" target="_blank" rel="noreferrer">
```

Comment:
Link really shouldn't be used for external links. 
It's designed to be used for internal routing inside the scope of the next app.